client/components/catalog/Card/index.vue

```diff
@@ -25,7 +25,7 @@
               <v-btn
                 v-on="on"
                 @click.stop="navigateTo('repository-info')"
-                color="blue-grey darken-1"
+                color="blue-grey lighten-2"
```

Comment:
We intentionally blended this icon a bit. Don't know about the @markolovric but being repeated on all cards now it looks a bit too much to me.